flink-streaming-java/src/main/java/org/apache/flink/streaming/api/operators/StreamingRuntimeContext.java

```diff
@@ -114,6 +116,10 @@ public void setKeyedStateStore(@Nullable KeyedStateStore keyedStateStore) {
         this.keyedStateStore = keyedStateStore;
     }
 
+    public void setAsyncKeyedStateStore(@Nullable AsyncKeyedStateStore asyncKeyedStateStore) {
```

Comment:
How about renaming `AsyncKeyedStateStore` to `KeyedStateStoreV2` or something? This is for new state v2 APIs instead of only for async states.

Reply:
Thanks for the suggestion.
Renamed to `KeyedStateStoreV2`.